M2/Macaulay2/packages/Varieties/doc-sheaves.m2

```diff
@@ -293,9 +570,60 @@ document {
     }
 -- TODO: should (module, SumOfTwists) return HH^0(F) or module F?
 
--- TODO: [(codim, CoherentSheaf), Generic]
+doc ///
+Node
+  Key
+    (isWellDefined, CoherentSheaf)
+  Headline
+    whether a coherent sheaf is well-defined
+  Usage
+    isWellDefined F
+  Inputs
+    F:CoherentSheaf
+  Outputs
+    :Boolean
+--  Description
+--    Text
+--    Example
+  SeeAlso
+    (isWellDefined, Variety)
+--    (isWellDefined, Module)
```

Comment:
What is actually checked here?

Reply:
Various data structure things (e.g. correct keys in the hash table), compatible rings, homogeneity of the module if the variety is projective, and maybe more in the future.